andela/activo-api
# #159192999 Delete a single center

#### What does this PR do?
>- Creates endpoint to delete a center
#### Description of Task to be completed?
>- **GIVEN** I have a valid authentication token
>- **WHEN** I `DELETE` to the `/centers/{<center_id>}` endpoint
>- **THEN** I will receive a confirmation of the successful deletion of a center
#### How should this be manually tested?
>- Run `git fetch origin ft-delete-center-159192999:ft-delete-center-159192999`, then run `git checkout ft-delete-center-159192999`
>- Run `flask run`
>- Send a DELETE request to the endpoint: `localhost:5000/api/v1/centers/{<center_id>}` 
>- You should receive a response, depending on the state of the centre concerned:

#### Any background context you want to provide?
>- N/A
#### What are the relevant pivotal tracker stories?
>- [159192999](https://www.pivotaltracker.com/story/show/159192999)
#### Blocked Tickets

#### Screenshots (if appropriate)
>- N/A
#### Questions:
>- N/A
#### Postman Collection
>- [Postman](https://www.getpostman.com/collections/4eae42b17b235eb0b982)
